### ISSUE TITLE
Paddy 2 crashes

### ISSUE THREAD

**Issue by mikeozornin** [DWilliames/paddy2-beta#12]
Sketch Version: 50 (54983).

When I select symbol made for paddy-1 paddy-2 (0.1) crashes.
It's quite complicated symbols (because of no nested symbols support in paddy), on simpler symbols it isn't crashing. 100% times.
Here is gist with crash log: https://gist.github.com/mikeozornin/52a634cfef2932bf93f2f55aed8eb95e
Need I share symbols?

When I insert into new document symbol from my sketch lib (paddy1-friendly) paddy2 crashes too.
Insert symbol (button) from lib.
Override text
Deselect button
Not 100% time.
Crashes with log: https://gist.github.com/mikeozornin/32c7cfe49339aa763ccfae75dd5552c8

When I make and override to the inserted symbol (mentioned in the first crash) sketch crashes too
Gist with log: https://gist.github.com/mikeozornin/33b12959db5ea78eb437e3c82295086a

**Comment by DWilliames** [comment]
Thanks @mikeozornin.
I just released a new version of the beta which should support Sketch 50 a little more.
The crash logs are really helpful.

Can you let me know if the new version fixes it at all. 
Cheers 👍

**Comment by mikeozornin** [comment]
I installed v0.2, but Sketch still crashing. Paddy do not crash Sketch when I'm using simple symbol (badge from library) but crash when I'm using complex symbol (buttons). 

When selecting symbol (not instance):
https://gist.github.com/mikeozornin/e21d43bf5ecb236fdab540e09936734a

When using override of a button symbol: https://gist.github.com/mikeozornin/ba7bc2d3d0d9355403fdaa4012eefd8c

I made a copy of our symbol library and removed almost all symbols. Here it is: https://sketch.cloud/s/drAvj. Maybe this will help to understand what's going on.

I can check not only release versions but some test builds too, if it needed.
